Cargo.toml

```diff
@@ -3059,6 +3059,18 @@ description = "Demonstrates volumetric fog and lighting"
 category = "3D Rendering"
 wasm = true
 
+[[example]]
+name = "anisotropy"
+path = "examples/3d/anisotropy.rs"
+doc-scrape-examples = true
+required-features = ["jpeg"]
```

Comment:
Should the `ktx2` feature be enabled too?

https://github.com/bevyengine/bevy/blob/main/Cargo.toml#L208

Reply:
It doesn't seem to require that for me. I think it's on by default?

Reply:
Just checked, seems `tonemapping_luts` implies `ktx2` and the former is enabled by default. Resolving this